cmake/aarch64.toolchain.cmake

```diff
@@ -0,0 +1,16 @@
+set(CMAKE_SYSTEM_NAME Linux)
```

Comment:
I wasn't sure how easy it would be to get it there, and it would be good to see it work here first. If it is easy to get it into core and then update core in realm-js we can do it that way.